Advertisements.DataAccess/Advertisements.DataAccess.csproj

```diff
@@ -83,12 +83,6 @@
     <Compile Include="Entities\IEntity.cs" />
     <Compile Include="Entities\PasswordRecovery.cs" />
     <Compile Include="Entities\Resource.cs" />
-    <Compile Include="Entities\Votes.cs" />
-    <Compile Include="Migrations\201708211048183_BaseMgr.cs" />
-    <Compile Include="Migrations\201708211048183_BaseMgr.Designer.cs">
-      <DependentUpon>201708211048183_BaseMgr.cs</DependentUpon>
-    </Compile>
-    <Compile Include="Migrations\Configuration.cs" />
```

Comment:
you merged migration in wrong way. Please leave these deleted migrations. Add your migrations after these.